src/In.ProjectEKA.HipService/DataFlow/Database/Migrations/20200204100834_AddingDataFlowRequestTable.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace In.ProjectEKA.HipService.DataFlow.Database.Migrations
+{
+    public partial class AddingDataFlowRequestTable : Migration
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4941971)